docs/partials/examples/ExampleBasic.vue

```diff
@@ -16,16 +16,32 @@
       | Must be between {{$v.age.$params.between.min}} and {{$v.age.$params.between.max}}
     pre
       | age: {{ $v.age }}
+    .form-group(v-bind:class="{ 'form-group--error': $v.cookies.$error }")
+      label.form__label Cookies
+      input.form__input(v-model.trim="cookies" @blur="$v.cookies.$touch()")
+    span.form-group__message(v-if='!$v.cookies.minValue')
+      | Must have minimum {{ $v.cookies.$params.minValue.min }}
+    pre
+      | cookies: {{ $v.cookies }}
+    .form-group(v-bind:class="{ 'form-group--error': $v.books.$error }")
+      label.form__label Books
+      input.form__input(v-model.trim="books" @blur="$v.books.$touch()")
+    span.form-group__message(v-if='!$v.books.maxValue')
+      | Must have maximum {{ $v.books.$params.maxValue.max }}
+    pre
+      | books: {{ $v.books }}
 </template>
 
 <script>
-import { required, minLength, between } from 'vuelidate/lib/validators'
+import { required, minLength, between, minValue, maxValue } from 'vuelidate/lib/validators'
 
 export default {
   data () {
     return {
       name: '',
-      age: 0
+      age: 0,
+      cookies: 0,
```

Comment:
I'd really like to avoid adding any more fields to the first example. It has to stay very approachable on a first sight, and to be honest it's already quite big.